gstd/gstd.c

```diff
@@ -200,12 +124,15 @@ main (gint argc, gchar * argv[])
   g_option_context_add_main_entries (context, entries, NULL);
 
   /* Initialize GStreamer */
-  gstreamer_group = gst_init_get_option_group ();
+  gstreamer_group = g_malloc (sizeof (GOptionGroup *));
+  gstd_manager_new (supported_ipcs, num_ipcs, &manager,
```

Comment:
Does `gstd_manager_new` take ownership of `gstreamer_group`? if so document it, otherwise you might be leaking it

Reply:
Comment added